<?xml version="1.0" encoding="UTF-8" ?>
<testsuites errors="0"
            failures="1"
            tests="1">

  <testsuite name="test"
    errors="0" failures="1" hostname="" id="" package="test" skipped="1"
    tests="1" time="5.759" timestamp="">
  
    <testcase name="default e2e tests" classname="test" time="5.759" assertions="2">

      <failure message="Testing if element [0;33m&lt;.hello&gt;[0m is present in 5000ms - expected [0;32m&#34;is present&#34;[0m but got: [0;31m&#34;not present&#34;[0m [0;90m(5170ms)[0m">    at Object.default e2e tests (C:\Users\hpaci\Desktop\timesheet-vue-front-master\tests\e2e\specs\test.js:9:15)
    at process._tickCallback (internal/process/next_tick.js:68:7)</failure>

    
      <failure message="    at Object.default e2e tests (C:\Users\hpaci\Desktop\timesheet-vue-front-master\tests\e2e\specs\test.js:9:15)">    at process._tickCallback (internal/process/next_tick.js:68:7)</failure>
    </testcase>
  

  

  
    
    <testcase
      name="example e2e test using a custom command" classname="test">
      <skipped />
    </testcase>
    
  
  </testsuite>
</testsuites>
